github-pullrequest-plugin/src/main/resources/com/github/kostyasha/github/integration/branch/GitHubBranchRepository/index.groovy

```diff
@@ -18,6 +19,15 @@ def makeBuildItem(def runs) {
     }
 }
 
+def escapeBranchName(def branchName) {
+   // escape anything that isn't alphanumeric
+   return StringEscapeUtils.escapeEcmaScript(branchName);
+}
+
+def makeRebuildId(def branchName) {
+    // replace anything that isn't alphanumeric so it's valid html
+    return ("rebuildResult" + branchName).replaceAll(/([^a-zA-Z0-9])/, '');
```

Comment:
commons has `CharSequenceTranslator`s, what is missing in `org.apache.commons.lang3.StringEscapeUtils#ESCAPE_ECMASCRIPT` ?